src/lib/containers/error/ErrorChip.tsx

```diff
@@ -0,0 +1,64 @@
+import { Avatar, Chip, Tooltip } from '@mui/material'
+import { ErrorOutlined, LockOutlined } from '@mui/icons-material'
```

Comment:
Not using IconSvg because it causes some strange spacing issues compared to just using the icons straight from MUI